api/app/controllers/spree/api/base_controller.rb

```diff
@@ -83,7 +101,8 @@ def load_user_roles
       end
 
       def unauthorized
-        render "spree/api/errors/unauthorized", status: 401 and return
+        render json: { error: I18n.t(:unauthorized, scope: "spree.api") },
+          status: 401 and return
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Use `&&` instead of `and`.